plugin-server/src/config/config.ts

```diff
@@ -206,3 +206,18 @@ export function overrideWithEnv(
     }
     return newConfig
 }
+
+export function buildIntegerMatcher(config: string, allowStar: boolean): ValueMatcher<number> {
+    // Builds a ConfigMatcher on a coma-separated list of values.
+    // Optionally, supports a '*' value to match everything
+    if (!config) {
+        return () => false
+    } else if (allowStar && config === '*') {
+        return () => true
+    } else {
+        const values = new Set(config.split(',').map((n) => parseInt(n)))
```

Comment:
`parseInt` returns a `Nan` on failure, we could decide to fail the parsing (and kill the pod) if that's the case, or silently ignore it like here